drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -711,11 +710,10 @@ static int mhdp_link_training(struct cdns_mhdp_device *mhdp,
 			   (mhdp->host.lanes_cnt & CDNS_SCRAMBLER) ? 0 :
 			   DP_LINK_SCRAMBLING_DISABLE);
 
-	cdns_mhdp_reg_write(mhdp, CDNS_DP_FRAMER_GLOBAL_CONFIG,
-			    CDNS_DP_NUM_LANES(mhdp->link.num_lanes) |
-			    CDNS_DP_DISABLE_PHY_RST |
-			    CDNS_DP_WR_FAILING_EDGE_VSYNC |
-			    (!video_mode ? CDNS_DP_NO_VIDEO_MODE : 0));
+	cdns_mhdp_reg_read(mhdp, CDNS_DP_FRAMER_GLOBAL_CONFIG, &reg32);
+	reg32 &= ~GENMASK(1, 0);
```

Comment:
Adding a
#define CDNS_DP_NUM_LANES_MASK GENMASK(1, 0)
and using that would make it bit more clear what the mask is for.  But this is more of nitpick. The register definitions as a whole would need a major rework.